libpod/image/image.go

```diff
@@ -847,22 +847,72 @@ func (i *Image) Dangling() bool {
 	return len(i.Names()) == 0
 }
 
+// User returns the image's user
+func (i *Image) User(ctx context.Context) string {
+	imgInspect, err := i.inspect(ctx, false)
+	if err != nil {
+		return ""
+	}
+	return imgInspect.Config.User
+}
+
+// StopSignal returns the image's StopSignal
+func (i *Image) StopSignal(ctx context.Context) string {
+	imgInspect, err := i.inspect(ctx, false)
+	if err != nil {
+		return ""
+	}
+	return imgInspect.Config.StopSignal
+}
+
+// WorkingDir returns the image's WorkingDir
+func (i *Image) WorkingDir(ctx context.Context) string {
+	imgInspect, err := i.inspect(ctx, false)
+	if err != nil {
+		return ""
+	}
+	return imgInspect.Config.WorkingDir
+}
+
+// Cmd returns the image's cmd
+func (i *Image) Cmd(ctx context.Context) []string {
+	imgInspect, err := i.inspect(ctx, false)
+	if err != nil {
+		return nil
+	}
+	return imgInspect.Config.Cmd
+}
+
+// Entrypoint returns the image's entrypoint
+func (i *Image) Entrypoint(ctx context.Context) []string {
+	imgInspect, err := i.inspect(ctx, false)
+	if err != nil {
+		return nil
+	}
+	return imgInspect.Config.Entrypoint
+}
+
+// Env returns the image's env
+func (i *Image) Env(ctx context.Context) []string {
+	imgInspect, err := i.imageInspectInfo(ctx)
+	if err != nil {
+		return nil
+	}
+	return imgInspect.Env
+}
+
 // Labels returns the image's labels
-func (i *Image) Labels(ctx context.Context) (map[string]string, error) {
+func (i *Image) Labels(ctx context.Context) map[string]string {
 	imgInspect, err := i.imageInspectInfo(ctx)
 	if err != nil {
-		return nil, nil
+		return nil
```

Comment:
Why are we just discarding errors? That doesn't seem sane

Reply:
The previous code was ignoring the errors. I thought there was a reason.

Reply:
Probably not? Can't think of one

Reply:
Fixed